assets/images/Vector.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
"Vector" é um nome muito genérico. Tenta usar um nome que descreva melhor o ícone

Reply:
Aliás, normalmente a gtente usa ícones no React de uma outra forma. 
É melhor incluir eles como componentes React, assim nós podemos customizar as propriedades CSS. Dá uma olhada nesse link aqui: https://dev.to/sgolovine/the-best-way-to-use-svg-icons-in-react-3ckm

Acho que pode colocar os ícones numa pasta 'assets/icons'.